changelog.md

```diff
@@ -67,5 +67,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Create ArrayInput
 - Add menu to see definition of component or attribute
 - Add model selection page + draw only one model in modelization page
+- Add carousel and image inside NewProjectTemplateDialog and ImportModelTemplateDialog respectively
```

Comment:
carousel of images ?

Reply:
in one case its a carousel of images (NewProjectTemplateDialog) and in the other it's only one image (no carousel in ImportModelTemplateDialog)

Reply:
changed